src/pricing.njk

```diff
@@ -6,148 +6,305 @@ meta:
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="mt-9 md:text-left items-left w-full mb-11 flex flex-row">
+                            <h2 class="self-end">$15</h2>
+                            <div class= "pl-1 text-sm leading-4 text-left self-end pb-1">
+                                <note class="note text-teal-100">/month</note>    
+                            </div>
+                    </div>
+                    <a class="md:self-end ff-btn ff-btn--primary uppercase align-baseline w-full" href="https://app.flowforge.com/account/create" onclick="capture('cta-join', {'position': 'primary'}, {'plan': 'cloud-starter'})">TRY FOR FREE</a> 
                 </div>
-                <div class="flex flex-col justify-between items-center text-center mb-4 sm:px-4 md:flex-row md:-mb-0 md:px-0 md:block md:text-left md:i md:items-left">
-                    <div class="flex flex-col items-center md:flex-row md:items-end md:justify-between">
-                        <div class="md:w-3/5">
-                            <h4 class="pt-4" style="display: inline-block;">Premium </h4>
-                            <label class="label">FlowFuse Cloud or On-Premise</label>
+            </div>
+    
+            <div class="pricing-tile md:h-full">
+                <div class="pricing-tile-background pb-6 md:grid flex-col">
+                    <div>
+                        <div class="flex flex-col items-center text-left md:flex-row md:-mb-0 md:px-0 md:block md:min-h-[133px]">
+                        <div class="flex flex-row justify-between mt-4 w-full gap-x-2">
+                            <h4 style="display:inline-block;">Team</h4>                        
+                        </div>
+                        <p class="mt-7">Allows <span class="inline font-normal">Node-RED developer teams</span> to benefit from the FlowForge platform.</p>
                         </div>
-                        <div class="annual-text h-full p-1 mt-1 md:w-2/5 md:ml-2">
-                            <a class="text" href='/contact-us'>Contact us for annual billing discounts</a>
+                        <div class="flex flex-col content-start features text-sm md:mt-3 mt-6 w-full">
+                            <note class="font-bold text-teal-100" style="display: inline-block;">Top Features:</note>
+                            <ul class="ff-checklist font-light text-white leading-7 mt-2">
+                                <li>Device management</li>
+                                <li>DevOps Pipelines</li>
+                                <li>Team collaboration</li>
+                            </ul>
                         </div>
+                    </div>    
+                    <div class="mt-9 mb-11 flex flex-row flex-wrap w-full gap-x-2">
+                            <div class="text-left flex flex-row">
+                            <h2 class="self-end">$19</h2>
```

Comment:
```suggestion
                            <h2 class="self-end">$25</h2>
```

Reply:
Hi @MarianRaphael I just updated that section :wink: